py/desispec/zfind/redmonster.py

```diff
@@ -58,14 +59,35 @@ def __init__(self, wave, flux, ivar, R=None, dloglam=1e-4):
         self.nwave = nwave
         self.nspec = nspec
 
+        #- Standardize objtype, converting ELG,LRG -> GALAXY, make upper case
+        templatetypes = set()
+        if objtype is None:
+            templatetypes = set(['GALAXY', 'STAR', 'QSO'])
+        else:
+            if isinstance(objtype, str):
+                objtype = [objtype,]
+                
+            objtype = [x.upper() for x in objtype]
+            for x in objtype:
+                if x in ['ELG', 'LRG']:
+                    templatetypes.add('GALAXY')
+                elif x in ['QSO', 'GALAXY', 'STAR']:
+                    templatetypes.add(x)
+                else:
+                    raise ValueError('Unknown objtype '+x)
+            
         #- list of (templatename, zmin, zmax) to fix
         self.template_dir = os.getenv('REDMONSTER_TEMPLATES_DIR')
-        self.templates = [
-            ('ndArch-spEigenStar-55734.fits', -0.005, 0.005),
-            ('ndArch-ssp_em_galaxy-v000.fits', 0.6, 1.6),
-            # ('ndArch-ssp_em_galaxy_quickdesi-v000.fits', 0.6, 1.6),
-            ('ndArch-QSO-V003.fits', 0.0, 3.5),
-        ]
+        self.templates = list()
+        for x in templatetypes:
+            if x == 'GALAXY':
+                self.templates.append(('ndArch-ssp_em_galaxy-v000.fits', 0.6, 1.6))
+            elif x == 'STAR':
+                self.templates.append(('ndArch-spEigenStar-55734.fits', -0.005, 0.005))
+            elif x == 'QSO':
+                self.templates.append(('ndArch-QSO-V003.fits', 0.0, 3.5))
+            else:
```

Comment:
I think we should go to z=2 by default.  There's a MgII 2800 (narrow) emission line that may occasionally yield a redshift.